parsec/core/gui/tr/parsec_en.po

```diff
@@ -1596,7 +1596,7 @@ msgid "TEXT_JOIN_ORG_INVALID_URL"
 msgstr "The link is invalid."
 
 msgid "TEXT_INVALID_URL"
-msgstr "The link is invalid."
+msgstr "The link is invalid.  (expected format : parsec://hote:port/organisation?action=action&..)"
```

Comment:
":" required a space before and after in French, but only after in English. You did the exact opposite. Could you also remove the dot?

`The link is invalid (expected format: parsec://hote:port/organisation?action=action&..)`